fineract-provider/src/main/java/org/apache/fineract/interoperation/service/InteropServiceImpl.java

```diff
@@ -354,7 +354,7 @@ public InteropQuoteResponseData createQuote(@NotNull JsonCommand command) {
 
         final BigDecimal fee;
         if (transactionType.isDebit()) {
-            fee = savingsAccount.calculateWithdrawalFee(request.getAmount().getAmount());
+            fee = savingsAccount.calculateTransactionFee(request.getAmount().getAmount());
```

Comment:
This must be an unrelated change, isn't it? If so, let's get rid of it.

Reply:
Done removed!